### DISCUSSION TITLE
Python 影像處理：OpenCV

### DISCUSSION THREAD

**Discussion by ching020721** [1101-GS4719-C/Python-Community-GS4719-C#912] [General]
OpenCV（Open Source Computer Vision）是一個跨平台的電腦視覺庫，包含許多電腦視覺相關演算處理的Open Source Library，而且支援多種開法語法。
若在進行影像處理實作必然會知道OpenCV，並且經常使用，這個網址有簡易介紹並教學如何使用opncv進行人臉辨識的練習。

https://ithelp.ithome.com.tw/articles/10251665

**Comment by manipulateed** [comment]
這篇分享的內容 淺顯易懂，透過訓練好的模型 能大大降低初學者的體驗難度。

**Comment by Kotori0501** [comment]
感謝分享，之前有修過圖像分析相關的課，我覺得非常有趣，也是用openCV 這個套件，有了這篇文章，又學到很多不一樣的了。

**Comment by dabbie109303501** [comment]
這也太酷了! 我之前有用過opencv，但只是單純拿來分析圖片上特定位置的顏色，沒想到還能那樣用

**Comment by Evan0979** [comment]
感覺戴口罩之後人臉辨識的難度就大大提高了,之後不知道能不能開發出只靠眼睛就能快速辨識人群的技術!!

**Comment by ruthchen1020** [comment]
感覺以後會有更多應用，像是可以達到用鏡頭理解3D世界的程度，這樣機器人可以取代人類更多工作了。

**Comment by evanchou1204** [comment]
雖然不是很精準，但用短短的程式碼做到辨別人臉已經很厲害了！

**Comment by CPCpyt** [comment]
感謝分享！內容淺顯易懂，而且感覺實務上會非常有趣。

**Comment by swerfg03joe** [comment]
這篇文章其實什麼理論都沒有講得很清楚，首先它是用內建的分類器去作出抓取人臉這個動作，根本不是人臉識別真要說的話只是人臉偵測，第二 分類器其實有相當多種，用的理論也是五花八門，如果是真的有心要做人臉辨識建議要去讀一下關於神經網路的課程

**Comment by BlahFish1006** [comment]
OpenCV真的很強大，只是我每次都卡在轉格式和影像數據串連上的問題過不去QQ
關關難過關關過，希望我有一天也能跟它變成熟悉的好朋友🥲

**Comment by WEI-CHEN-CHUN** [comment]
網站介紹的很淺顯，OpenCV感覺相當強大

**Comment by s96015welson** [comment]
這篇文章的教學只是著重於人臉偵測，就是找出圖中有幾張人臉以及其位置，後續還需要將人臉裁切下來後，再使用神經網路才可以去判斷出每張人臉是誰。此外，opencv的分類器還有很多，像是瞳孔偵測等等，通常是下載opencv就會附在裡面了，可以直接在CascadeClassifier資料夾裡看到有哪些附帶的分類器。
